lighthouse-extension/app/src/extension-entry.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 const lighthouse = require('../../../lighthouse-core/index');
-const background = require('./lighthouse-background');
+const background = require('./devtools-entry');
```

Comment:
it's unfortunate to have to keep calling this `background`, but I can't think of a better name right now :)

Reply:
if I move `getDefaultConfigForCategories`  & `getDefaultCategories` to it's own file called one of them below I could rename the variable which gives more meaning to it.
- `config.js`,
- `helpers/config.js`
- `runOptions.js`
- `lighthouseOptions.js`
- ...

Reply:
yeah, I think that makes sense, but I think we should just wait on it (at least until LR shakes out and we can fully see the shape of these files for the medium-term future).